CodeFormatter/clClangFormatLocator.cpp

```diff
@@ -25,22 +24,14 @@ bool clClangFormatLocator::Locate(wxString& clangFormat)
     nameOptions.Add("clang-format-3.4");
     nameOptions.Add("clang-format-3.3");
 
+    wxFileName file;
     for(size_t i = 0; i < nameOptions.GetCount(); ++i) {
-        fnClangFormat.SetFullName(nameOptions.Item(i));
-        if(fnClangFormat.Exists()) {
-            clangFormat = fnClangFormat.GetFullPath();
+        if (clFindExecutable(nameOptions.Item(i), file)) {
+            clangFormat = file.GetFullPath();
             return true;
         }
     }
     return false;
-#else
-    wxFileName fnClangFormat(clStandardPaths::Get().GetBinaryFullPath("codelite-clang-format"));
```

Comment:
This part should be restored. on Windows/OSX `codelite-clang-foramt` is always located under `clStandardPaths::Get().GetBinaryFullPath`